rd-net/RdFramework/Impl/SocketWire.cs

```diff
@@ -522,9 +523,12 @@ protected virtual bool AcceptHandshake(Socket socket)
     public class Client : Base
     {
       public Client(Lifetime lifetime, IScheduler scheduler, int port, string? optId = null) :
-        this(lifetime, scheduler, new IPEndPoint(IPAddress.Loopback, port), optId) {}
+        this(lifetime, scheduler, EndPointWrapper.CreateIpEndPoint(IPAddress.Loopback, port), optId) {}
+      
+      public Client(Lifetime lifetime, IScheduler scheduler, string path, string? optId = null) :
+        this(lifetime, scheduler, EndPointWrapper.CreateUnixEndPoint(path), optId) {}
```

Comment:
_(Minor)_ Since this an unusual use case, I'd prefer to have something other than a `string` in here. Let's maybe make a separate struct, like this?
```csharp
public struct UnixSocketConnectionParams
{
  public string Path { get; set; }
}
```
and then
```
public Client(Lifetime lifetime, IScheduler scheduler, UnixSocketConnectionParams connectionParams, string? optId = null)
```

WDYT?